pkg/ngctl/cmd/version/version.go

```diff
@@ -0,0 +1,123 @@
+/*
+Copyright 2021 Vesoft Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package version
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/kubernetes/pkg/apis/core"
+
+	"github.com/vesoft-inc/nebula-operator/pkg/label"
+	cmdutil "github.com/vesoft-inc/nebula-operator/pkg/ngctl/cmd/util"
+	"github.com/vesoft-inc/nebula-operator/pkg/ngctl/cmd/util/ignore"
+	operatorversion "github.com/vesoft-inc/nebula-operator/pkg/version"
+)
+
+const (
+	versionExample = `
+		# Print the cli and nebula operator version
+		ngctl version
+`
+)
+
+type (
+	// Options is a struct to support version command
+	Options struct {
+		ClientOnly bool
+
+		kubeCli kubernetes.Interface
+		genericclioptions.IOStreams
+	}
+)
+
+// NewOptions returns initialized Options
+func NewOptions(ioStreams genericclioptions.IOStreams) *Options {
+	return &Options{
+		IOStreams: ioStreams,
+	}
+}
+
+// NewCmdVersion returns a cobra command for fetching versions
+func NewCmdVersion(f cmdutil.Factory, ioStreams genericclioptions.IOStreams) *cobra.Command {
+	o := NewOptions(ioStreams)
+	cmd := &cobra.Command{
+		Use:     "version",
+		Short:   "Print the cli and nebula operator version",
```

Comment:
use Long: "Print the cli and nebula operator version", it's better than Long == ""